.github/promote-images.yml

```diff
@@ -17,12 +17,11 @@
     workspace-node-18: "TIMESTAMP_TAG"
     workspace-node-20: "TIMESTAMP_TAG"
     workspace-node-22: "TIMESTAMP_TAG"
-    workspace-python: "TIMESTAMP_TAG"
```

Comment:
Why we need to remove `workspace-python`? (I thought images without specified versions will point to latest stable)

Reply:
@mustard-mh that's what we do for other images like `workspace-node` indeed, but we have not actually been updating it since [last August](https://github.com/gitpod-io/workspace-images/pull/1169), so this is just a cleanup. We can re-introduce it in the dazzle.yaml in a follow-up if we need to, but this is just syncing what we build to what we push.

Reply:
> @mustard-mh that's what we do for other images like workspace-node indeed

From README.md, we still have `workspace-node`, it's not removed. 

Btw, `workspace-python` is [documented](https://www.gitpod.io/docs/introduction/languages/python) in our docs as well (after this PR, we'll not maintain it anymore but users with this image configured don't know it
<img width="831" alt="image" src="https://github.com/user-attachments/assets/10a85116-0e7a-4364-ac34-9fca412cb173">